src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/10 Page/on.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'on(event, handler)'
+excerpt: 'Browser module: page.on method'
+---
+
+Allows the user to define an event handler function that will be executed every time the specified event is emitted.
+
+| Parameter       | Type   | Default | Description                                                                                                                                                                                                                           |
+|-----------------|--------|---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| event        | string  | `''`    |  Event to attach handler to. Currently, only `'console'` event is supported.                                                                                                               |
+| handler        | function  | `null`    |  A function to be called every time the specified event is emitted.                                                                                                               |
+
+
+<Blockquote mod="warning" title="">
+
+When using `page.on` method, the page has to be explicitly [closed](/javascript-api/k6-experimental/browser/page/close/) for the iteration to be able to finish.
+
+</Blockquote>
+
+
+### on('console')
```

Comment:
Should this be an `Events` section containing a table? Or should we keep it like this as there is only one supported event by now?

Reply:
Just my two cents, I think switching it to a table might be better. As a heading, it took me a second to understand what exactly this section was referring to, and I think switching it to a table (even if it's just one item) might make it easier to parse the page as a whole.

Reply:
I tried the table approach in [f069738](https://github.com/grafana/k6-docs/pull/1308/commits/f069738ba4c705ec0880d211d8aca6b4431f3562). Let me know what do you think.

Reply:
I like this better! Thanks for making the change. 🙇